local_manifest.xml

```diff
@@ -8,13 +8,13 @@
 
   
   <!-- surnia stuff -->
-  <project path="device/motorola/surnia" name="oreo-surnia/android_device_motorola_surnia" revision="lineage-15.0" remote="gitstuff" />
-  <project path="device/motorola/msm8916-common" name="oreo-surnia/android_device_motorola_msm8916-common" revision="lineage-15.0-alberto" remote="gitstuff"/>
-  <project path="kernel/motorola/msm8916" name="oreo-surnia/android_kernel_motorola_msm8916"  revision="lineage-15.0-alberto" remote="gitstuff"/>
+  <project path="device/motorola/osprey" name="AospExtended-Devices/device_motorola_osprey" revision="8.x" remote="gitstuff" />
+  <project path="device/motorola/msm8916-common" name="AospExtended-Devices/device_motorola_msm8916-common" revision="8.x" remote="gitstuff"/>
```

Comment:
The common and vendor uses the AEX-Devices repos, just merged it on lineage-15.0-alberto